source/reference/minio-mc/mc-license-update.rst

```diff
@@ -10,6 +10,10 @@
 
 .. mc:: mc license update
 
+.. important:: 
+
+   ``mc license update`` requires :ref:`MinIO Client <minio-client>` version ``RELEASE.2023-11-20T16-30-59Z`` or newer.
```

Comment:
I think we'll need to suggest a server version here too. Most users/customers want a minimum viable target.

Reply:
Added.

Reply:
No, there is absolutely no need to upgrade `minio` server and it will be a very unfair ask for customers to upgrade their servers. Please revert this change.

Reply:
I thought the standard was that the MinIO Client and the MinIO Server must stay in sync?
Otherwise, the MinIO Client can show options that won't work for the MinIO Server.

Reply:
mc must be upgraded if server is upgraded not the other way around.

Yes if they upgrade mc we will ask them to upgrade MinIO if they need new functionality but it is not a hard requirement unless you want the new feature in mc to work.